src/public.js

```diff
@@ -23,29 +23,30 @@
  *
  */
 
-import Vue from 'vue';
-import userapp from './UserApp';
-import VModal from 'vue-js-modal';
+import Vue from 'vue'
+import UserApp from './UserApp'
+import VModal from 'vue-js-modal'
 
-Vue.use(VModal);
+Vue.use(VModal)
 
-Vue.prototype.t = t;
-Vue.prototype.n = n;
-Vue.prototype.OC = OC;
-Vue.prototype.OCA = OCA;
+Vue.prototype.t = t
+Vue.prototype.n = n
+Vue.prototype.OC = OC
+Vue.prototype.OCA = OCA
 
-
-const hasToken = (document.getElementById('sharingToken') !== null);
-const isPasswordProtected = (document.getElementById('password-submit') !== null);
+const hasToken = (document.getElementById('sharingToken') !== null)
+const isPasswordProtected = (document.getElementById('password-submit') !== null)
 
 if (hasToken && !isPasswordProtected) {
-	$('body').prepend($('<div>').attr('id', 'terms_of_service_confirm'));
+	const tofc = document.createElement('div')
+	tofc.id = 'terms_of_service_confirm'
+	document.body.prepend(tofc)
+
 	new Vue({
```

Comment:
I need help here @skjnldsv 

This throws:
```
/home/nickv/Nextcloud/19/server/appsshipped/terms_of_service/src/public.js
  45:2  error  Do not use 'new' for side effects  no-new
```

but prepending `export default` like in all the other places didn't work:
```
/home/nickv/Nextcloud/19/server/appsshipped/terms_of_service/src/public.js
  45:2  error  Parsing error: 'import' and 'export' may only appear at the top level

  43 | 	document.body.prepend(tofc)
  44 | 
> 45 | 	export default new Vue({
     | 	^
  46 | 		el: '#terms_of_service_confirm',
  47 | 		data: {
  48 | 			source: 'public',
```

Reply:
Probably better to just use `// eslint-disable-next-line`.

Reply:
It's because it's nested into a if
export/import needs to be at the root level of the app :)

Just disable the `no new` rule :)

Reply:
Thanks!